### PR TITLE
automated PR: update CoC and legalese

### PR DESCRIPTION
**Pull request by ericchiang**

### CODE REVIEW

File: code-of-conduct.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+project maintainers commit themselves to fairly and consistently applying these
+principles to every aspect of managing this project. Project maintainers who do
+not follow or enforce the Code of Conduct may be permanently removed from the
+project team.
+
+This code of conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community.
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting a project maintainer, Brandon Philips
```
Comment (srenatus) [inline code-of-conduct.md]: Is this accurate? Or is [`MAINTAINERS`](https://github.com/coreos/dex/blob/master/MAINTAINERS) out-of-date?

Comment (ericchiang) [reply code-of-conduct.md]: Yep, MAINTAINERS is still up to date.

Comment (xiang90) [reply code-of-conduct.md]: @srenatus the maintainers of CoC of CoreOS org projects is different from the technical maintainers of the actual projects. so you see the difference.

**Comment by ericchiang** [comment]
cc @diegs for approval

**Pull request merged**